### ISSUE TITLE
MHW need Weapen Info

### ISSUE THREAD

**Issue by SakuraSayaka** [HunterPie/HunterPie#345]
MHW needs Weapon Info, just like in v1, otherwise it can't replace the Player Hud that comes with the game, because there is no Weapon Info in v2

**Comment by WhaleK233** [comment]
agree, I want v2 like v1 to show weapon info because I need to know how much time my buff left

**Comment by WhaleK233** [comment]
agree, I want v2 like v1 to show weapon info because I need to know how much time my wirebug buff left

**Comment by A-K-T** [comment]
It's been six months since this issue was posted and weapon details are still missing. The entire player HUD widget is effectively unusable outside a handful of weapons because using it doesn't allow you to track weapon-specific resources.
